src/backend/marsha/core/models/file.py

```diff
@@ -0,0 +1,94 @@
+"""This module holds the model for files and derivated resources."""
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from ..defaults import PENDING, STATE_CHOICES
+from .account import User
+from .base import BaseModel
+from .playlist import Playlist
+
+
+class BaseFile(BaseModel):
```

Comment:
Shouldn't it also be the base model for the Video model?

Reply:
Yes it should be great but I have error when I try to generate a new migration file.